client/src/components/Card/Card.css

```diff
@@ -115,7 +115,7 @@
 
 .Card__discover-link {
   text-align: center;
-  margin: auto 20px auto 20px;
+  margin: 50px 20px 50px 20px;
```

Comment:
@dakom0 is changing the CSS going to have any unintended consequences for any other pages that are using this component?